projects/ffmpeg.cmake

```diff
@@ -1,5 +1,16 @@
 ########################################
 # ffmpeg
+
+# FFmpeg requires YASM
+find_program(yasm_location yasm)
+if (${yasm_location} STREQUAL "yasm_location-NOTFOUND")
+  message(FATAL_ERROR "\n"
+    "yasm not found -- FFmpeg can't be built. install on linux:\n"
+    "  apt install yasm\n"
+    "  yum install yasm  # requires epel-release\n")
+  return()
+endif()
+
```

Comment:
👍 Thanks for adding this, I had wanted to add something to check that YASM was intsalled when I originally put in FFmpeg and openH264, but never got around to it.